lab1.ipynb

```diff
@@ -0,0 +1,921 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "540df231-1756-4546-a320-f165b0a9f1c2",
+   "metadata": {},
+   "source": [
+    "# Lab 1 "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "a9f7e6cc-e97d-4c41-8d63-b3eed6c283a8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipy.stats"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b82c861f-4425-4963-acb3-0608a14f7198",
+   "metadata": {},
+   "source": [
+    "## A little statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bfc5f06-4d95-4970-a2ec-29846327eb14",
+   "metadata": {},
+   "source": [
+    "### &nbsp;&nbsp;&nbsp;1. Converting a probability into a 'sigma'."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1efb1b58-8e40-4a58-91e5-19b430474dfe",
+   "metadata": {},
+   "source": [
+    "#### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; A. Reading (no code needed)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "643a2008-32d7-4780-8f0d-3046f81406e1",
+   "metadata": {},
+   "source": [
+    "In my reading of what a normal distribution is, there were several main points covered. Frist of all, it is also known as a Gaussian Distribution. It is a distribution centered on a mean. The width of the curve is based on a variable known as sigma. 68% of all data points in a Gaussian Distribution will be within plus or minus 1 sigma of the mean, while 95% will be within 2, and 99.7% within 3. The more sigma from the mean that one goes, the closer to 100% that one approaches, however, since the Gaussian only approaches zero at the infinites, the only way to get 100% of all possible outcomes is to go infinite sigma away. The integral of any normal distribution will always equal 1, regardless of mean that it is centered on or standard deviation (sigma squared)."
```

Comment:
Throughout the entire notebook. You have consistently good descriptions. Very clear